cfe_internal/enterprise/federation/federation.cf

```diff
@@ -565,9 +590,9 @@ bundle agent exported_data
 
   commands:
     am_feeder.!am_paused::
-      "/bin/bash"
+      "/bin/bash" -> { "CFE-951" }
         arglist => {"$(cfengine_enterprise_federation:config.bin_dir)/dump.sh"},
-        contain => default:in_shell,
+        contain => default:in_shell_dir("/tmp"),
```

Comment:
why in `/tmp` ? isn't this run as root here?

Reply:
WHY? Because biggest hammer.
It is run as root.
ACK,